### DISCUSSION TITLE
Problems with: Make binaries executable

### DISCUSSION THREAD

**Discussion by juferprada97** [SysBioChalmers/RAVEN#526] [Help me!] [answered]
I have the following problem when installing Raven in matlab (Liinux64 - ubuntu) with the "make binaries executables".

line 1: syntax error near unexpected token `('
/bin/bash: -c: line 1: `chmod +x
"/home/jufer/Apps/Raven/RAVEN/software/blast+/blastp".(mexa64|mexglx|mexmaci64|mac|bat)'  
> In checkInstallation>makeBinaryExecutable (line 338)
In checkInstallation (line 107) 
Warning: Failed to make /home/jufer/Apps/Raven/RAVEN/software/blast+/makeblastdb executable: /bin/bash:
-c: line 1: syntax error near unexpected token `('
/bin/bash: -c: line 1: `chmod +x
"/home/jufer/Apps/Raven/RAVEN/software/blast+/makeblastdb".(mexa64|mexglx|mexmaci64|mac|bat)'  
> In checkInstallation>makeBinaryExecutable (line 338)
In checkInstallation (line 107) 
Warning: Failed to make /home/jufer/Apps/Raven/RAVEN/software/cd-hit/cd-hit executable: /bin/bash: -c:
line 1: syntax error near unexpected token `('
/bin/bash: -c: line 1: `chmod +x
"/home/jufer/Apps/Raven/RAVEN/software/cd-hit/cd-hit".(mexa64|mexglx|mexmaci64|mac|bat)'  
> In checkInstallation>makeBinaryExecutable (line 338)
In checkInstallation (line 107) 
Warning: Failed to make /home/jufer/Apps/Raven/RAVEN/software/diamond/diamond executable: /bin/bash:
-c: line 1: syntax error near unexpected token `('
/bin/bash: -c: line 1: `chmod +x
"/home/jufer/Apps/Raven/RAVEN/software/diamond/diamond".(mexa64|mexglx|mexmaci64|mac|bat)'  
> In checkInstallation>makeBinaryExecutable (line 338)
In checkInstallation (line 107) 
Warning: Failed to make /home/jufer/Apps/Raven/RAVEN/software/hmmer/hmmbuild executable: /bin/bash: -c:
line 1: syntax error near unexpected token `('
/bin/bash: -c: line 1: `chmod +x
"/home/jufer/Apps/Raven/RAVEN/software/hmmer/hmmbuild".(mexa64|mexglx|mexmaci64|mac|bat)'  
> In checkInstallation>makeBinaryExecutable (line 338)
In checkInstallation (line 107) 
Warning: Failed to make /home/jufer/Apps/Raven/RAVEN/software/hmmer/hmmsearch executable: /bin/bash:
-c: line 1: syntax error near unexpected token `('
/bin/bash: -c: line 1: `chmod +x
"/home/jufer/Apps/Raven/RAVEN/software/hmmer/hmmsearch".(mexa64|mexglx|mexmaci64|mac|bat)'  
> In checkInstallation>makeBinaryExecutable (line 338)
In checkInstallation (line 107) 
Warning: Failed to make /home/jufer/Apps/Raven/RAVEN/software/GLPKmex/glpkcc executable: /bin/bash: -c:
line 1: syntax error near unexpected token `('
/bin/bash: -c: line 1: `chmod +x
"/home/jufer/Apps/Raven/RAVEN/software/GLPKmex/glpkcc".(mexa64|mexglx|mexmaci64|mac|bat)'  
> In checkInstallation>makeBinaryExecutable (line 338)
In checkInstallation (line 107) 
Warning: Failed to make /home/jufer/Apps/Raven/RAVEN/software/libSBML/TranslateSBML executable:
/bin/bash: -c: line 1: syntax error near unexpected token `('
/bin/bash: -c: line 1: `chmod +x
"/home/jufer/Apps/Raven/RAVEN/software/libSBML/TranslateSBML".(mexa64|mexglx|mexmaci64|mac|bat)'  
> In checkInstallation>makeBinaryExecutable (line 338)
In checkInstallation (line 107) 
Warning: Failed to make /home/jufer/Apps/Raven/RAVEN/software/libSBML/OutputSBML executable: /bin/bash:
-c: line 1: syntax error near unexpected token `('
/bin/bash: -c: line 1: `chmod +x
"/home/jufer/Apps/Raven/RAVEN/software/libSBML/OutputSBML".(mexa64|mexglx|mexmaci64|mac|bat)'  
> In checkInstallation>makeBinaryExecutable (line 338)
In checkInstallation (line 107) 
Warning: Failed to make /home/jufer/Apps/Raven/RAVEN/software/mafft/mafft-linux64/mafft executable:
/bin/bash: -c: line 1: syntax error near unexpected token `('
/bin/bash: -c: line 1: `chmod +x
"/home/jufer/Apps/Raven/RAVEN/software/mafft/mafft-linux64/mafft".(mexa64|mexglx|mexmaci64|mac|bat)'  
> In checkInstallation>makeBinaryExecutable (line 338)
In checkInstallation (line 107) 
Warning: Failed to make /home/jufer/Apps/Raven/RAVEN/software/mafft/mafft-mac/mafft executable:
/bin/bash: -c: line 1: syntax error near unexpected token `('
/bin/bash: -c: line 1: `chmod +x
"/home/jufer/Apps/Raven/RAVEN/software/mafft/mafft-mac/mafft".(mexa64|mexglx|mexmaci64|mac|bat)'  
> In checkInstallation>makeBinaryExecutable (line 338)
In checkInstallation (line 107) 
Fail

**Comment by edkerk** [comment, marked as answer]
This is caused by a bug in the code. Issue #527 has been opened. PR #528 will solve this. Will be part of 2.8.7 release (#524).
